backend/app/services/source_family_profiles.py

```
@@ -0,0 +1,194 @@
+    required_version_fields=(
+        "dataset_contract_version",
+        "schema_snapshot_version",
+        "execution_policy_version",
+        "connector_profile_version",
+        "dialect_profile_version",
+    ),
+    connector=ConnectorProfileRequirements(
+        profile_id="mysql.readonly.planned.v1",
+        owner="backend",
+        read_only_posture="required",
+        secret_reference_pattern="safequery/business/mysql/<source_id>/reader",
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to argument: "secret_reference_pattern"

(S106)